src/mqtt_t.cpp

```diff
@@ -320,6 +343,177 @@ bool mqttPublishLoopFromTxBuffer()
   return true;
 }
 
+void mqttpublishHADiscovery()
+{
+
+        // MQTT Home Assistant Discovery
+        // -----------------------------------
+
+        char buf[512];
```

Comment:
The stack buffer of the `loopTask()` should be big enough, right?
If I looked at the right place it has 8192 bytes, so this should not be an issue.